.beta-goreleaser.yml

```diff
@@ -0,0 +1,39 @@
+# This is an example goreleaser.yaml file with some sane defaults.
+# Make sure to check the documentation at http://goreleaser.com
+
+builds:
+- main: ./cli
+  env:
+  - CGO_ENABLED=0
+archives:
+  - id: archive
+    replacements:
+      darwin: Darwin
+      linux: Linux
+      windows: Windows
+      386: i386
+      amd64: x86_64
+    format_overrides:
+      - goos: windows
+        format: zip
+checksum:
+  name_template: 'checksums.txt'
+snapshot:
+  name_template: "{{ .Tag }}-next"
+changelog:
+  sort: asc
+  filters:
+    exclude:
+    - '^docs:'
+    - '^test:'
+brews:
+  -
+    github:
+      owner: stelligent
+      name: homebrew-tap
+    commit_author:
+      name: goreleaserbot
+      email: goreleaser@stelligent.com
+    folder: Formula/beta
+release:
+  prerelease: auto
```

Comment:
This allows a tag with a name ending in `-STRING` to be labeled as a `Pre-Release`.